frontend/tests/axe.spec.ts

```diff
@@ -1,14 +1,15 @@
 import { AxeBuilder } from "@axe-core/playwright";
-import { expect, test } from "@playwright/test";
+import { expect, test, type Page } from "@playwright/test";
 import analyses from "@/fixtures/analyses.json" with { type: "json" };
 import { log } from "./util";
 
 log();
 
-/** pages to test */
-const paths = [
+/** pages to test */ 
+//exported path Also to be used in lighthouse.spec.ts
+export const paths = [
```

Comment:
Instead of duplicating this between the Axe and Lighthouse test, you can just extract the paths to a separate `paths.ts` file, and import the paths from both tests. The error you got from importing the paths directly from the Axe test file is probably a limitation of the Playwright testing suite. You probably can't cross reference objects or any other imports between test files. But hoisting the paths list into a separate file seems to solve it.